nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/AbstractTestCommentNotification.java

```diff
@@ -192,7 +192,7 @@ public void shouldNotifyWithTheRightCommentedDocument() {
             assertEquals(2, mails.size());
             String expectedMailContent = getExpectedMailContent(replyDocumentModel, commentedDocumentModel,
                     expectedEvent);
-            assertEquals(expectedMailContent, getMailContent(mails.get(0)));
+            assertEquals(expectedMailContent, getMailContent(mails.get(1)));
```

Comment:
This features revealed an error in the test after flattening test methods. The first mail was checked instead of the 2nd. Since content was strictly identical, test was successful.
In this ticket that doesn't cut it anymore since replies have different template.
I will fix it directly in the autosubscribe backport.